modules/test/playwright/tests/object-web/utils/getRandomObjectField.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {getRandomInt} from '../../../utils/getRandomInt';
+
+interface Props {
+	fieldsQuantity: number;
+}
+
+export default function getRandomObjectField({
+	fieldsQuantity,
+}: Props): ObjectField[] {
+	const objectFields = [];
+
+	for (let i = 1; i <= fieldsQuantity; i++) {
+		const objectField = {
+			DBType: 'String',
+			businessType: 'Text',
+			externalReferenceCode: 'customText' + getRandomInt(),
+			indexed: true,
+			indexedAsKeyword: false,
+			indexedLanguageId: '',
+			label: {en_US: 'customText' + getRandomInt()},
+			listTypeDefinitionId: 0,
```

Comment:
Hey @vngoliveira, could you check if the property `listTypeDefinitionId` is really necessary?